src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -1147,9 +1387,52 @@ public Bounds getBounds() {
 
         } // end for polygon
 
-        return new Bounds(
+        bounds= new Bounds(
                 new Vector3d(minX, minY, minZ),
                 new Vector3d(maxX, maxY, maxZ));
+        return bounds;
+    }
+    /**

```

Comment:
Here are a few more helper functions for accessing the cached bounds object. They flatten out the get of the min and max X,Y,Z bounds, making the use syntax easier to understand.